src/Workspaces/Core/Portable/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -360,7 +421,8 @@ public enum ModifierKindEnum
             IsStatic,
             IsAsync,
             IsReadOnly,
-            IsConst
+            IsConst,
+            Ignore // This matches any modifiers
```

Comment:
this feels wrong.  paging @dpoeschl .   If we want to match any modifier then why not have either an optional list of modifiers to match, or make no modifiers mean this, or just have a list with all modifiers.